dashboard/app/models/course.rb

```diff
@@ -27,7 +27,8 @@ class Course < ApplicationRecord
 
   FAMILY_NAMES = [
     CSD = 'csd'.freeze,
-    CSP = 'csp'.freeze
+    CSP = 'csp'.freeze,
+    TEST = 'ui-test-course'.freeze
```

Comment:
Probably, I just added `ScriptConstants:: COURSE_FAMILY_NAMES` recently and didn't know about `Course::FAMILY_NAMES` 🤦‍♀️ 

I was mostly looking focused on removing course and script names from ScriptConstants since they're what needs to be updated each year. Seems like a good thing to consolidate but not high on my priorities either.

Reply:
I think I have cleaned this up now.